Formula/ssh-audit.rb

```diff
@@ -8,7 +8,6 @@ class SshAudit < Formula
 
   bottle :unneeded
 
```

Comment:
Sorry, need to fix the audit failure too. You can try this, but `brew audit --strict --fix ssh-audit` might be a safer bet.

```suggestion
```